### PR TITLE
[Doc Feature][Conversational bots in GCCH][3334399]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1ac4539affdd59dbe126738cb688c8b9e923a944/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 90 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/896421de-1aae-4db7-a662-3c6ad4cc0270) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2cee364e5726c514010d6a860c9c8c1b29fe39cf/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 90 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c67d0eb4-8060-4b01-9c85-c71be60cb09d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2cee364e5726c514010d6a860c9c8c1b29fe39cf/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e195814a-70e1-4b6f-bf5d-bcccb5d7e6b5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2cee364e5726c514010d6a860c9c8c1b29fe39cf/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eb34a2d2-383f-4c02-9ebd-37cf090ae987) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/295a174d00297b453b20bb6708068373cf1a3d8d/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 92 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b61a4e38-2893-49fb-ac6c-db8f6c16c43d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/295a174d00297b453b20bb6708068373cf1a3d8d/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/57db304a-0036-4a25-9087-099d830c7fe3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/295a174d00297b453b20bb6708068373cf1a3d8d/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/385ed769-be2b-4ca2-a63a-4187d5d5c3eb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9eb40087e44137bff07178d5a0cb517abd214679/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 92 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bab7ea30-191d-4bfa-9e8a-65c0c677fe17) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/9eb40087e44137bff07178d5a0cb517abd214679/msteams-platform%2Fmstdd-landing.yml) | 89 | 1 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/69b361c7-bd2e-4c4b-9835-063a5fd4b011) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9eb40087e44137bff07178d5a0cb517abd214679/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/731ae2e5-93ef-40b4-9fb4-8ced7799db64) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/20c2db3d756b6840448eefff423321a518b2ba77/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 92 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/02381e8e-a519-4047-ad90-62f9d2d646b7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/20c2db3d756b6840448eefff423321a518b2ba77/msteams-platform%2Fmstdd-landing.yml) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fa817545-3cbe-4d82-ae44-ea394fd29429) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/20c2db3d756b6840448eefff423321a518b2ba77/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c4d0ee06-6b4c-4435-b1c1-8482dd1a0660) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3a6fed00f3177529867a3ea68436bd76b2584fd0/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 92 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3696ab17-e16b-4ee0-9d91-8a16eb567c70) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/app-fundamentals-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3a6fed00f3177529867a3ea68436bd76b2584fd0/msteams-platform%2Fconcepts%2Fapp-fundamentals-overview.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b985549b-cd25-4855-9cb3-f1013f120586) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3a6fed00f3177529867a3ea68436bd76b2584fd0/msteams-platform%2Fmstdd-landing.yml) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7f935270-6509-404d-bb69-1ab6672440b6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3a6fed00f3177529867a3ea68436bd76b2584fd0/msteams-platform%2Fwhats-new.md) | 94 | 38 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/926090ef-e28a-47aa-8d07-ae8e1ec9a61e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by rm3f — APPROVED** [review]

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Meetings enable collaboration, partnership, informed communication, and shared f
 <br>
 You can create scenes for meetings, provide notifications to users, populate in-meeting dialogs, and more with meeting app extensibility.
 
-Third-party and line-of-business apps built for meetings and calls are available in Government Community Cloud (GCC) but aren't available for GCC-High and Department of Defense (DOD) tenants.
+Third-party and line-of-business apps built for meetings and calls are available only in Government Community Cloud (GCC) and GCC-High environments but aren't available for Department of Defense (DOD) environment.
```
Comment (rm3f) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:73]: Here we say DoD is not available, but the checkboxes below show as available. If this PR is for both the gcch and dod releases, then it should say "Third-party and line-of-business apps built for meetings and calls are available in Government Community Cloud (GCC),  GCC-High and Department of Defense (DOD),"
If the PR is for GCCH only, then the checkbox for dod should be removed below.

Comment (rm3f) [reply msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:73]: disregard as this PR only applies to gcch.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/20a95f2e3a50ba93646036b477e6bc72e9be5abc/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 92 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9df1f39b-7eef-4e75-8ad7-c5bfb427056b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/app-fundamentals-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/20a95f2e3a50ba93646036b477e6bc72e9be5abc/msteams-platform%2Fconcepts%2Fapp-fundamentals-overview.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/49f7f3f8-f3e4-4d92-8031-edc33dd96d8b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/20a95f2e3a50ba93646036b477e6bc72e9be5abc/msteams-platform%2Fmstdd-landing.yml) | 91 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/311de41d-2e3e-4028-853c-da961e69bf6f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/20a95f2e3a50ba93646036b477e6bc72e9be5abc/msteams-platform%2Fwhats-new.md) | 94 | 40 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eddf4dc4-ad8c-4f12-aaa9-5e457b26315e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by rm3f — DISMISSED** [review]

**Review by juna-gupta — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed21eddc319c2afcc8f222603acc4bc16cf58ffe/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 89 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b9330379-e3bf-4e25-89ac-ad842f73a1d0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/app-fundamentals-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed21eddc319c2afcc8f222603acc4bc16cf58ffe/msteams-platform%2Fconcepts%2Fapp-fundamentals-overview.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7ddc1011-40bb-4bb1-91c8-933c963fbe7f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/ed21eddc319c2afcc8f222603acc4bc16cf58ffe/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ce054098-820f-4cc4-b5dd-ba31fb1c09c0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed21eddc319c2afcc8f222603acc4bc16cf58ffe/msteams-platform%2Fwhats-new.md) | 94 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/60ce6978-95d4-4d23-8bf6-61701a3ef99b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f05674fa77cbdf2a9ab37cef02442f5f21ddf22a/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 89 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/19a299a0-61bf-4928-b240-273f829c9414) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/app-fundamentals-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f05674fa77cbdf2a9ab37cef02442f5f21ddf22a/msteams-platform%2Fconcepts%2Fapp-fundamentals-overview.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cc390bcf-414b-414c-a1b1-73d380afe168) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/f05674fa77cbdf2a9ab37cef02442f5f21ddf22a/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0ee3e1b1-81b5-423e-8fba-b2ca9be22fe7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f05674fa77cbdf2a9ab37cef02442f5f21ddf22a/msteams-platform%2Fwhats-new.md) | 94 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/85807bf4-7579-4dcb-86fb-b962343fc18a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2804e220a1c53ace54abee35b853bcd94d18dfe4/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 89 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7d16e0da-0532-4b05-bf89-b7973b17b976) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/app-fundamentals-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2804e220a1c53ace54abee35b853bcd94d18dfe4/msteams-platform%2Fconcepts%2Fapp-fundamentals-overview.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ed849005-90e6-4312-8e9c-41845fec3f6d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2804e220a1c53ace54abee35b853bcd94d18dfe4/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3c792663-6ffb-4c00-8609-7278e8ef33b1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2804e220a1c53ace54abee35b853bcd94d18dfe4/msteams-platform%2Fwhats-new.md) | 94 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/208a34ca-cf30-4cf2-9d4d-b085425ada96) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/86af9f537c374fccf6623ba034ba1f7b6900b401/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 89 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/41fd7a54-0a86-4d20-8e2f-c138280f1991) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/app-fundamentals-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/86af9f537c374fccf6623ba034ba1f7b6900b401/msteams-platform%2Fconcepts%2Fapp-fundamentals-overview.md) | 93 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0709caa7-31ea-45d7-b8ff-f44f398bee4b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/86af9f537c374fccf6623ba034ba1f7b6900b401/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e46c6add-0f46-4e78-8104-fc92910f18cd) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/86af9f537c374fccf6623ba034ba1f7b6900b401/msteams-platform%2Fwhats-new.md) | 94 | 42 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b5849f61-097c-481d-a45a-fcb7c0bf748a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**